certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2031 @@
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            }
+          }
+        }
+      }
+    },
+    "/UpdateTicket": {
+      "post": {
+        "summary": "Update a Ticket",
+        "description": "Update a Ticket",
+        "operationId": "UpdateTicket",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Actions.UpdateTicket"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response from Update Ticket",
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            }
+          }
+        }
+      }
+    },
+    "/ticket/{id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Response from Get One Ticket",
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            },
+            "headers": {
+              "accept": {
+                "description": "accept",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Get Ticket",
+        "description": "Get One Ticket",
+        "operationId": "GetOneTicket",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Enter Tikit ID",
+            "x-ms-summary": "Tikit ID"
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Define oData select",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "Define oData select"
+          }
+        ]
+      }
+    },
+    "/FileUpload/{id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "string",
+                  "description": "data"
+                },
+                "error": {
+                  "type": "string",
+                  "description": "error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get File(s) Attached to a Ticket",
+        "description": "Get File(s) Attached to a Ticket",
+        "operationId": "GetFileAttached",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Tikit ID",
+            "description": "Enter Tikit ID"
+          }
+        ]
+      }
+    },
+    "/Comment": {},

```

Comment:
@TikitTeam Empty operations can be removed as they would not add any value to your connector.

Reply:
Fixed!